website/docs/docs/build/metrics-overview.md

```diff
@@ -75,7 +75,7 @@ metrics:
     label: The display name for your metric. This value will be shown in downstream tools. ## Required
     filter: |                             ## Optional            
       {{  Dimension('entity__name') }} > 0 and {{ Dimension(' entity__another_name') }} is not
-      null
+      null and {{ Metric('metric_name', group_by=['entity']) }} > 5
```

Comment:
```suggestion
      null AND {{ Metric('metric_name', group_by=['entity']) }} > 5
```

Reply:
Would also update this to say `entity_name` instead of `entity` (for consistency with what's below)